src/app/modules/notification/components/notifications/notifications.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
it's normal to have the notification commit of `Marie` in your PR or it's a rebase problem ?